src/languages/en.ts

```diff
@@ -2315,7 +2315,7 @@ export default {
             importedFromAccountingSoftware: 'The taxes below are imported from your',
         },
         emptyWorkspace: {
-            title: 'Create a workspace',
+            title: 'Create a new workspace',
```

Comment:
Actually I think we can just keep the existing copy that we had before, sorry for the misleading mockup! Let's undo this change.